jetstream/tests/test_config.py

```diff
@@ -25,6 +25,22 @@ def test_configs_from(self):
         assert configs_collection.configs is not None
         assert len(configs_collection.configs.configs) == len(loader.configs.configs)
 
+    def test_configs_from_null(self):
+        loader = _ConfigLoader()
+        base_collection = loader.with_configs_from(
+            ["https://github.com/mozilla/metric-hub/tree/main/jetstream"]
+        )
+        new_collection = base_collection.with_configs_from(None)
+        assert new_collection == base_collection
+
+    def test_configs_from_empty(self):
+        loader = _ConfigLoader()
+        base_collection = loader.with_configs_from(
+            ["https://github.com/mozilla/metric-hub/tree/main/jetstream"]
+        )
+        new_collection = base_collection.with_configs_from(())
```

Comment:
I think this should be empty list not tuple.
```suggestion
        new_collection = base_collection.with_configs_from([])
```

Reply:
I think the default value for click.option when multiple is set is [an empty tuple](https://github.com/pallets/click/blob/main/src/click/core.py#L2510)

Reply:
Which is what's passed as the value for `private_config_repos` [here](https://github.com/mozilla/jetstream/blob/main/jetstream/cli.py#L1165)

Reply:
Ah interesting, fair enough! I was looking at the param type for the function signature, silly me.